<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="5" passed="4">
  <reporter-output>
    <line>
      <![CDATA[Screen Shot Taken]]>
    </line>
    <line>
      <![CDATA[Screen Shot Taken]]>
    </line>
    <line>
      <![CDATA[Screen Shot Taken]]>
    </line>
    <line>
      <![CDATA[Screen Shot Taken]]>
    </line>
  </reporter-output>
  <suite name="LoginTCSuite" duration-ms="29206" started-at="2020-07-31T00:12:37Z" finished-at="2020-07-31T00:13:06Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="29206" started-at="2020-07-31T00:12:37Z" finished-at="2020-07-31T00:13:06Z">
      <class name="com.busyqa.batch16.uitests.testCases.TC01_LoginTest">
        <test-method status="PASS" signature="setUp(java.lang.String, java.lang.String)[pri:0, instance:com.busyqa.batch16.uitests.testCases.TC01_LoginTest@27976390]" name="setUp" is-config="true" duration-ms="13002" started-at="2020-07-31T05:42:37Z" finished-at="2020-07-31T05:42:50Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[qa]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="failed_TC1()[pri:0, instance:com.busyqa.batch16.uitests.testCases.TC01_LoginTest@27976390]" name="failed_TC1" duration-ms="384" started-at="2020-07-31T05:42:50Z" finished-at="2020-07-31T05:42:50Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:286)
	at org.testng.Assert.assertEquals(Assert.java:296)
	at com.busyqa.batch16.uitests.testCases.TC01_LoginTest.failed_TC1(TC01_LoginTest.java:175)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:348)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:38)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:382)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- failed_TC1 -->
        <test-method status="PASS" signature="loginTest01()[pri:0, instance:com.busyqa.batch16.uitests.testCases.TC01_LoginTest@27976390]" name="loginTest01" duration-ms="4914" started-at="2020-07-31T05:42:50Z" finished-at="2020-07-31T05:42:55Z">
          <reporter-output>
            <line>
              <![CDATA[Screen Shot Taken]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest01 -->
        <test-method status="PASS" signature="loginTest02()[pri:0, instance:com.busyqa.batch16.uitests.testCases.TC01_LoginTest@27976390]" name="loginTest02" duration-ms="2042" started-at="2020-07-31T05:42:55Z" finished-at="2020-07-31T05:42:57Z">
          <reporter-output>
            <line>
              <![CDATA[Screen Shot Taken]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest02 -->
        <test-method status="PASS" signature="loginTest03()[pri:0, instance:com.busyqa.batch16.uitests.testCases.TC01_LoginTest@27976390]" name="loginTest03" duration-ms="1884" started-at="2020-07-31T05:42:57Z" finished-at="2020-07-31T05:42:59Z">
          <reporter-output>
            <line>
              <![CDATA[Screen Shot Taken]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest03 -->
        <test-method status="PASS" signature="loginTest04()[pri:0, instance:com.busyqa.batch16.uitests.testCases.TC01_LoginTest@27976390]" name="loginTest04" duration-ms="6564" started-at="2020-07-31T05:42:59Z" finished-at="2020-07-31T05:43:06Z">
          <reporter-output>
            <line>
              <![CDATA[Screen Shot Taken]]>
            </line>
          </reporter-output>
        </test-method> <!-- loginTest04 -->
        <test-method status="PASS" signature="endTest()[pri:0, instance:com.busyqa.batch16.uitests.testCases.TC01_LoginTest@27976390]" name="endTest" is-config="true" duration-ms="347" started-at="2020-07-31T05:43:06Z" finished-at="2020-07-31T05:43:06Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- com.busyqa.batch16.uitests.testCases.TC01_LoginTest -->
    </test> <!-- Test -->
  </suite> <!-- LoginTCSuite -->
</testng-results>
